client/cli/src/params/network_params.rs

```diff
@@ -146,6 +146,10 @@ pub struct NetworkParams {
 		verbatim_doc_comment
 	)]
 	pub sync: SyncMode,
+
+	/// Maximum number of blocks per request.
```

Comment:
should we provide some guidance on what the good number should be? i.e. "the optimal number depends on ..."